src/qibolab/_core/dummy/platform.py

```diff
@@ -38,6 +39,10 @@ def create_dummy() -> Platform:
         pump_name: DummyLocalOscillator(address="0.0.0.0"),
     }
 
-    return Platform.load(
-        path=FOLDER, instruments=instruments, qubits=qubits, couplers=couplers
-    )
+    return Hardware(instruments=instruments, qubits=qubits, couplers=couplers)
+
+
+def create_dummy() -> Platform:
+    """Create a dummy platform using the dummy instrument."""
+    hardware = create_dummy_hardware()
+    return Platform.load(path=FOLDER, **vars(hardware))
```

Comment:
You could even change the return type to `Hardware` directly, and delegate the loading to the same `create_platform()` machinery handling the other platforms.

I.e. turn
https://github.com/qiboteam/qibolab/blob/00164e3d1680f5341b3fc57a8fb7a8688a73c7db/src/qibolab/_core/platform/load.py#L75-L80
into
```py  
path = (
    Path(__file__).parents[1] / "dummy"
    if name == "dummy"
    else _search(name, _platforms_paths())
)
```

Reply:
However, I just noticed `create_dummy` is not part of the public interface...

It's true that usage in tests is limited
https://github.com/search?q=repo%3Aqiboteam%2Fqibolab+create_dummy+path%3A%2F%5Etests%5C%2F%2F&type=code
and most likely even direct usage outside Qibolab (because `create_platform("dummy")` is fully equivalent). But it would be breaking.

We may consider deprecating, and maintain a single access point to the dummy platform. But we need to schedule these for 0.3

Reply:
> However, I just noticed `create_dummy` is not part of the public interface...

I think you meant to say "it is part of the public interface". In this case yes, this is why I wanted to maintain its behavior the same, even though I agree it is not needed to be public. Shall we then open an issue for 0.3 about this?

Reply:
Correct, the *not* was just a mistake.

> Shall we then open an issue for 0.3 about this?

Maybe yes. It is a tiny change in any case, but it saves some (small) duplication. It should be fine as a small issue.

Reply:
> However, I just noticed `create_dummy` is not part of the public interface...
> 
> It's true that usage in tests is limited https://github.com/search?q=repo%3Aqiboteam%2Fqibolab+create_dummy+path%3A%2F%5Etests%5C%2F%2F&type=code and most likely even direct usage outside Qibolab (because `create_platform("dummy")` is fully equivalent). But it would be breaking.
> 
> We may consider deprecating, and maintain a single access point to the dummy platform. But we need to schedule these for 0.3

#1154